x-pack/plugins/security/server/routes/authentication/index.ts

```diff
@@ -16,7 +16,6 @@ export function createCustomResourceResponse(body: string, contentType: string,
     body,
     headers: {
       'content-type': contentType,
-      'cache-control': 'private, no-cache, no-store',
```

Comment:
**note**: the default cache control headers will apply to these custom responses as well. The existing SAML and OIDC functional tests will continue to assert this.